docs/source/docs/examples/poseest.md

```diff
@@ -0,0 +1,206 @@
+         :language: c++
+         :lines: 38-46
+         :linenos:
+         :lineno-start: 38
+
+    .. tab-item:: Python
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-python-examples/poseest/robot.py
+         :language: python
+         :lines: 54-56
+         :linenos:
+         :lineno-start: 54
+
+```
+
+### Simulating the Camera
+
+First, we create a new {code}`VisionSystemSim` to represent our camera and coprocessor running PhotonVision, and moving around our simulated field.
+
+```{eval-rst}
+.. tab-set::
+
+    .. tab-item:: Java
+       :sync: java
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/poseest/src/main/java/frc/robot/Vision.java
+         :language: java
+         :lines: 65-69
+         :linenos:
+         :lineno-start: 65
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/include/Vision.h
+         :language: c++
+         :lines: 49-52
+         :linenos:
+         :lineno-start: 49
+
+    .. tab-item:: Python
+
+       # TODO
+
+```
+
+Then, we add configure the simulated vision system to match the camera system being simulated.
+
+```{eval-rst}
+.. tab-set::
+
+    .. tab-item:: Java
+       :sync: java
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/poseest/src/main/java/frc/robot/Vision.java
+         :language: java
+         :lines: 69-82
+         :linenos:
+         :lineno-start: 69
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/include/Vision.h
+         :language: c++
+         :lines: 53-65
+         :linenos:
+         :lineno-start: 53
+
+    .. tab-item:: Python
+
+       # TODO
+```
+
+
+### Updating the Simulated Vision System
+
+During simulation, we periodically update the simulated vision system.
+
+```{eval-rst}
+.. tab-set::
+
+    .. tab-item:: Java
+       :sync: java
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/poseest/src/main/java/frc/robot/Robot.java
+         :language: java
+         :lines: 114-132
+         :linenos:
+         :lineno-start: 114
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/cpp/Robot.cpp
+         :language: c++
+         :lines: 95-109
+         :linenos:
+         :lineno-start: 95
+
+    .. tab-item:: Python
+
+       # TODO
+```
+
+The rest is done behind the scenes.
```

Comment:
This page feels incomplete, either a gif, summary, comprehensive code block, link to relevant docs something should be here to show what has actually been accomplished. Its a lot of very high level concepts but lacks some closure that ties pose estimation, simulation, and photonvision together.

Reply:
gif incoming. tomorrow.

Reply:
done